src/handlers/package_handler.js

```diff
@@ -128,8 +127,7 @@ async function postPackages(req, res) {
 
   if (exists.ok) {
     // The package exists.
-    error.publishPackageExists(res);
-    logger.httpLog(req, res);
+    common.packageExists(req, res);
```

Comment:
```suggestion
    await common.packageExists(req, res);
```

Not an issue of your implementation, just adding this while noticing that it's missing.